modules/compatibility/src/test/java/org/apache/ignite/compatibility/testframework/util/MavenUtils.java

```diff
@@ -42,6 +42,9 @@
     /** */
     private static final String GG_MVN_REPO = "http://www.gridgainsystems.com/nexus/content/repositories/external";
 
+    /** */
+    private static final String LOCAL_PROXY_MAVEN_SETTINGS = "~/.m2/local-proxy.xml";
```

Comment:
The path to settings file should be taken from environmental variable of the same name.

Reply:
I made this the default value, in case the environment variable is not declared.